### PR TITLE
Add information on Social Preview Image

### PR DESCRIPTION
**Pull request by MaxUlysse**
- Add _Social Preview Image_ section in `markdown/developers/adding_pipelines.md `
- Remove information about Zenodo in ` markdown/developers/release_checklist.md` as it has to be done only on the first release

### CODE REVIEW

File: markdown/developers/adding_pipelines.md [hunk 1]
```diff
@@ -219,6 +219,24 @@ You can check that all of these settings are done correctly by referring to your
...
+* You might need to install the google font [Maven Pro](https://fonts.google.com/specimen/Maven+Pro)
+* Update the pipeline name
+* If, and only if pipeline name is too long then center the logo
+* Update the pipeline description
+* Save as `assets/nf-core-<pipeline-name>_social_preview.svg`
+* Export as `assets/nf-core-<pipeline-name>_social_preview.png`
+* Upload it on the nf-core pipeline GitHub repository in the _Social Preview_ section in the _Settings_ tab
+
+Unfortunately, these steps cannot be automated, as the description of the pipeline could take one or two lines.
+Line break should not
```
Comment (ewels) [inline markdown/developers/adding_pipelines.md]: ..?

File: markdown/developers/adding_pipelines.md [hunk 2]
```diff
@@ -219,6 +219,24 @@ You can check that all of these settings are done correctly by referring to your
...
 This reports the status of various checks and also has the option of fixing errors for you via the GitHub API.
 
+### Social Preview Image
+
+From the [nf-core/logos repository](https://github.com/nf-core/logos), download the [social preview image template](https://github.com/nf-core/logos/blob/master/nf-core-logos/social_preview_image_template.svg), and follow these instructions:
+
+* Open the `social_preview_image_template.svg` file using [Inkscape](https://inkscape.org/) or any other vector graphics editor
+* You might need to install the google font [Maven Pro](https://fonts.google.com/specimen/Maven+Pro)
+* Update the pipeline name
+* If, and only if pipeline name is too long then center the logo
```
Comment (ewels) [inline markdown/developers/adding_pipelines.md]:
```suggestion
* If the pipeline name goes beyond the apple, centre the logo in the image
```

File: markdown/developers/adding_pipelines.md (hunk 1 above)
Comment (MaxUlysse) [inline markdown/developers/adding_pipelines.md]:
```suggestion
Line break should look nice and be readable
```

File: markdown/developers/adding_pipelines.md [hunk 3]
```diff
@@ -219,6 +219,24 @@ You can check that all of these settings are done correctly by referring to your
...
+* You might need to install the google font [Maven Pro](https://fonts.google.com/specimen/Maven+Pro)
+* Update the pipeline name
+* If the pipeline name goes beyond the apple, centre the logo in the image
+* Update the pipeline description
+* Save as `assets/nf-core-<pipeline-name>_social_preview.svg`
+* Export as `assets/nf-core-<pipeline-name>_social_preview.png`
+* Upload it on the nf-core pipeline GitHub repository in the _Social Preview_ section in the _Settings_ tab
+
+Unfortunately, these steps cannot be automated, as the description of the pipeline could take one or two lines.
+Line break should look nice and be readable
```
Comment (MaxUlysse) [inline markdown/developers/adding_pipelines.md]:
```suggestion
Line break should look nice and be readable.
```

File: markdown/developers/adding_pipelines.md (hunk 1 above)
Comment (MaxUlysse) [reply markdown/developers/adding_pipelines.md]: I must not have saved the right version of this phrase...

**Pull request merged**